docker-berp/Dockerfile

```diff
@@ -0,0 +1,51 @@
+# Builds a docker image used for building most projects in this repo. It's
+# used both by contributors and CI.
+#
+FROM mcr.microsoft.com/dotnet/sdk:6.0
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+RUN apt-get update \
+    && apt-get install --assume-yes \
+    locales
```

Comment:
I've used this package before and its primary use is switching/checking things on a CLI, i.e. timezone

Are we needing to do this? Or would just using an interactive console / script suffice?

Reply:
Yes: dotNet expects UTF8 locales to be installed. There's no need to change it on the cli, but dotNet needs it for its operation.